### DISCUSSION TITLE
Rival car audio

### DISCUSSION THREAD

**Discussion by Tenome** [emoose/OutRun2006Tweaks#223] [Suggestions (Multiplayer)]
### Describe the feature you'd like to see added

I'm gonna assume this isn't possible, but is there anything in the code that might let you hear other racers' audio? Even disregarding online, you don't hear rival racers either, so I'm guessing the player car is the only one programmed to produce sfx. I have a horn, and no one to honk at.

**Comment by Outrunner24** [comment]
> I'm guessing the player car is the only one programmed to produce sfx

Not 100% true: you hear the "passing" noise from other cars and trucks honk at you when you're playing their minigame in career mode. But yeah, if the code doesn't have that, you can't just activate something that does not exist.
